js/app.js

```diff
@@ -1,2 +1,89 @@
-$(function () {
+$(document).ready(function () {
+  $('#submit-btn').click(() => {
+
+    refreshSearch()
+
+    const city = $('#city-type').val()
+
+    getWeatherByCity(city)
+
+  })
+
+
+  function getWeatherByCity (city) {
+    axios.get(`http://api.openweathermap.org/data/2.5/weather?q=${city}&APPID=1c739a9713a0919a3024a089c3b027df`)
```

Comment:
@kcycle it's recommended that you use `https` instead of `http` whenever available. The use of `http` here may subject you to a CORS error.